src/actions/sendCallLogs.tsx

```diff
@@ -0,0 +1,24 @@
+export default async (roomName, log) => {
```

Comment:
missing typing
`(roomName: string, log: Record<string, unknown>)`

Reply:
Thanks for the feedback @ntchou! For the moment I only solved lint errors and left `noImplicitAny: false` to avoid having to fix everything at once. I thought it will be easier to strong type everything in an incremental way.

Reply:
Once we merge this one, don't hesitate to make PRs to type the parts of the code you know the type of :)

Reply:
You're right ;)